learnify/mobile/test/verification_screen_test.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:learnify/core/widgets/buttons/action_button.dart';
+import 'package:learnify/features/auth/verification/constants/widget_keys.dart';
+import 'package:learnify/features/auth/verification/view/verification_screen.dart';
+
+import 'test_helpers.dart';
+
+void main() {
+  testWidgets("Test verification screen", (WidgetTester tester) async {
+    final VerificationScreen verificationScreen =
+        VerificationScreen(email: "example@gmail.com");
```

Comment:
Using a realistic email can be better, please change this. Other than that, all seem okay. Thanks.

Reply:
Done